extensions/WAYLIVES/SVG/1.0.1.js

```diff
@@ -0,0 +1,282 @@
+// Name: SVG
+// ID: WRsvg
+// Description: Create SVG elements.
+// By: WAYLIVES <https://scratch.mit.edu/users/WAYLIVES/>
+// License: MIT
+
+// Version: 1.0.1
+
+(function (Scratch) {
+  "use strict";
+
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error("svg");
+  }
+
+  class svg {
+    getInfo() {
+      return {
+        id: "svg",
+        name: Scratch.translate("SVG"),
+        color1: "#9823FF",
+        color2: "#7C2DC1",
+        color3: "#2D3548",
+        docsURI: "https://extensions.turbowarp.org/WAYLIVES/SVG",
+        blocks: [
+          "---",
+
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: Scratch.translate("The basis of SVG:"),
```

Comment:
```suggestion
            text: Scratch.translate("The basics of SVG"),
```

I can't tell if this is supposed to be basics or basis

also, you should remove the colon from every single one of your label blocks